build.xml

```diff
@@ -991,6 +991,33 @@ This target has been renamed to "snapshot-bundle". Please call "snapshot-bundle"
 		<ant dir="util-taglib" inheritAll="false" target="restore-latest-artifact-version" />
 	</target>
 
+	<target name="setup-elasticsearch">
+		<if>
+			<resourcecount count="0" when="gt">
+				<fileset
+					dir="${project.dir}/modules/apps/portal-search-elasticsearch6"
+					includes="**/.lfrbuild-portal"
+				/>
+			</resourcecount>
+			<then>
+				<ant antfile="${project.dir}/build-test-elasticsearch6.xml" inheritAll="false" target="setup-elasticsearch" />
+			</then>
+			<elseif>
+				<resourcecount count="0" when="gt">
+					<fileset
+						dir="${project.dir}/modules/apps/portal-search-elasticsearch7"
+						includes="**/.lfrbuild-portal"
+					/>
+				</resourcecount>
+				<then>
+					<ant antfile="${project.dir}/build-test-elasticsearch7.xml" inheritAll="false" target="setup-elasticsearch" />
+
+					<ant antfile="${project.dir}/build-test-elasticsearch7.xml" inheritAll="false" target="deploy-elasticsearch7" />
```

Comment:
Hi @petershin,
I do not think we need this line.
The deploy task is going to deploy ES7 related modules, if we already hve the build trigger in ES7 folder, it will be deployed automatically. 

Tina.

Reply:
@tinatian - Appreciate the review, updated the logic